developer-docs-site/docs/concepts/coin-and-token/fungible-asset.md

```diff
@@ -0,0 +1,444 @@
+---
+title: "Fungible Asset"
+id: "fungible-asset"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Fungible Asset
+
+Fungible assets (FA) are an essential part of the Aptos ecosystem, as they enable the creation and transfer of fungible
+units, which can represent different things, such as currency, shares, material in games, or any other type of asset.
+Fungible assets can be used to build decentralized applications that require a token economy, such as decentralized
+exchanges or gaming platforms.
+
+[Fungible asset module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move)
+provides a standard, typesafe framework for assets with fungibility.
+
+In this standard, fungible assets are stored in `Object<FungibleStore>` that has a specific amount of units, which
+can be transferred, burned, or minted. Fungible assets are units that are interchangeable with others of the same
+metadata. The standard is built upon object model so all the resources defined here are included in object resource
+group and stored inside objects.
+
+## Structures
+
+### Metadata Object
+
+FA metadata defines the type of FA. A metadata is defined in Move as:
+
+```rust
+#[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+struct Metadata has key {
+/// Optional tracking of the current supply of the fungible asset.
+supply: Option<Supply>,
+/// Name of the fungible metadata, i.e., "USDT".
+name: String,
+/// Symbol of the fungible metadata, usually a shorter version of the name.
+/// For example, Singapore Dollar is SGD.
+symbol: String,
+/// Number of decimals used for display purposes.
+/// For example, if `decimals` equals `2`, a balance of `505` coins should
+/// be displayed to a user as `5.05` (`505 / 10 ** 2`).
+decimals: u8,
+}
+```
+
+:::tip
+This is similar to `CoinType` in coin standard but with richer info. The most significant difference is that even two
+exactly the same metadata object denote two types instead of one.
+:::
+
+A Coin uses the `CoinType` to support re-usability of the Coin framework for distinct Coins. For example, `Coin<A>`
+and `Coin<B>` are two distinct coins.
+
+### Fungible Asset and Fungible Store
+
+Since metadata is specified, FA could be defined as follows:
+
+```rust
+struct FungibleAsset {
+    metadata: Object<Metadata>,
+    amount: u64,
+}
+```
+
+Simple, right? An object representing the type and the amount of units held. It is noted that it does not have has
+no abilities, hence it can only be alive in execution runtime. In other words, it must be consumed and cannot be
```

Comment:
it can only be alive in execution runtime => It can be passed from one function to another but has to be deposited back into a fungible store at the end of the transaction